src/sql/parts/query/editor/messagePanel.ts

```diff
@@ -116,6 +119,66 @@ export class MessagePanel extends ViewletPanel {
 				this.state.collapsed = !this.isExpanded();
 			}
 		});
+		this.controller.onKeyDown = (tree, event) => {
+			if (event.ctrlKey) {
+				let context: IMessagesActionContext = {
+					selection: null,
+					tree: this.tree,
+					selectAllMessages: null
+				};
+				// Ctrl + A for select all
+				if (event.code === 'KeyA') {
+					let allMessages = this.copySelectAllMessagesText();
+					let selectAllAction = instantiationService.createInstance(SelectAllMessagesAction, this);
+					context.selectAllMessages = allMessages;
+					selectAllAction.run(context);
```

Comment:
does this actually highlight the selection?

Reply:
Yes it did highlight it then.